### ISSUE TITLE
Run a11y test over deprecation-app

### ISSUE THREAD

**Issue by serenaf** [ember-learn/deprecation-app#81]
See https://github.com/ember-learn/guides-app/issues/46 as example and follow ;)

**Comment by takshch** [comment]
This issue is too old. Is this issue still exits?
If yes, assign me to this issue.

**Comment by ijlee2** [comment]
Hi, @takshch. Thanks for taking an interest in helping out. I think it's a good idea to audit accessibility in tests. However, there are currently 2 soft-blocking issues.

One, installing `ember-a11y-testing v4` will result in an error,

```
ember-a11y-testing has the following unmet peerDependencies:
  * @ember/test-helpers: `^2.0.0`; it was resolved to `1.7.2`
```

because `ember-qunit v4.6` still uses version 1.x of `@ember/test-helpers`. We could install `v3.x` of `ember-a11y-testing` to see how much work will be needed in fixing a11y issues.

Two, the app may be redesigned some time in the future so the work that you may put into may be gone.

Before we decide whether to continue with this issue, I'd like to hear what your thoughts are on these two matters. Thanks again!

**Comment by MinThaMie** [comment]
Close this for now, because we need to come up with a strategy for this topic across all our learning repos :)
